dockstore/dockstore-ui2
# SEAB-5318: Created execution tab in UI for metrics

**Description**
This PR adds the Execution tab in workflows to show its metrics. This tab can be accessed by the feature flag `metrics`.

When workflow has no metrics data available
![Screenshot from 2023-04-26 17-42-21](https://user-images.githubusercontent.com/61166764/234709679-764d7887-49ca-4d65-975b-a5665174d97b.png)

Available partners can be toggled through the dropdown and if workflow does not have any validations data the bottom table gets hidden
![Screenshot from 2023-04-26 17-42-42](https://user-images.githubusercontent.com/61166764/234709745-cefeb072-631e-4d92-932a-d48c2ab83cfc.png)

If most recent run is valid 
![Screenshot from 2023-04-27 09-53-33](https://user-images.githubusercontent.com/61166764/234884988-24c16e20-b7b5-4625-967d-58d88d556684.png)

If most recent run is not valid 
![Screenshot from 2023-04-26 17-43-58](https://user-images.githubusercontent.com/61166764/234709786-deec32e0-e55c-415f-a0f8-5844a0a6e013.png)

**Review Instructions**

1.  Go to [this workflow](https://qa.dockstore.org/workflows/github.com/dockstore-testing/tooltester-wes-testing/nontrivial:main?tab=info) and add the feature flag `metrics`.
2. Verify that the executions tab is visible with its corresponding metrics data. Also verify that the partners dropdown toggles through the various available metrics data correctly.
3.  Repeat with this [workflow](https://qa.dockstore.org/workflows/github.com/gatk-workflows/seq-format-conversion/BAM-to-Unmapped-BAM:3.0.0?tab=info) but check that the validations table below is hidden (since there are no available validations data)
4. Workflows with no metrics data available should have the banner stating so. [example](https://qa.dockstore.org/workflows/github.com/iwc-workflows/sars-cov-2-ont-artic-variant-calling/COVID-19-ARTIC-ONT:main?tab=info)

**Issue**
https://ucsc-cgl.atlassian.net/browse/SEAB-5352

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!
